mapeditor/Animation.cpp

```diff
@@ -686,6 +689,44 @@ std::shared_ptr<QImage> Animation::getImage(size_t frame, size_t group, bool ver
 	return nullptr;
 }
 
+void Animation::exportBitmaps(const bfs::path& path, bool prependResourceName) const
+{
+	if (images.empty())
+	{
+		logGlobal->error("Nothing to export, animation is empty");
+		return;
+	}
+
+	boost::filesystem::path actualPath = path / "Sprites" / name;
+	boost::filesystem::create_directories(actualPath);
+
+	size_t counter = 0;
+
+	for (const auto& groupPair : images)
+	{
+		size_t group = groupPair.first;
+
+		for (const auto& imagePair : groupPair.second)
+		{
+			size_t frame = imagePair.first;
+			const auto img = imagePair.second;
+
+			boost::format fmt("%d_%d.png");
```

Comment:
please use QString - you can avoid type conversion

Reply:
I left it like this because its the same code as in CAnimation.cpp

Reply:
converted to QString